scripts/setup_wince.sh

```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
+
+# #############################################################
```

Comment:
please put this in echo so it also print out when this script runs

Reply:
Done.